packages/client/src/components/SpeechBubble.jsx

```diff
@@ -0,0 +1,55 @@
+import { theme } from '../theme.jsx';
+import { Box, Typography } from '@mui/material';
+
+export function SpeechBubbleRight({ text }) {
+  return SpeechBubble(false, { text });
+}
+
+export function SpeechBubbleLeft({ text }) {
+  return SpeechBubble(true, { text });
+}
+
+function SpeechBubble(isLeft, { text }) {
+  return (
+    <Box sx={{ position: 'relative' }}>
+      <svg
+        style={{ position: 'absolute', zIndex: -1 }}
+        width="100%"
+        height="100%"
+        viewBox="0 0 370 170"
+        preserveAspectRatio="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        {isLeft ? (
+          <g transform="scale(-1, 1) translate(-370,0)">
+            <SpeechBubblePath />
+          </g>
+        ) : (
+          <SpeechBubblePath />
+        )}
+      </svg>
+      <Typography variant="h5" sx={{ padding: '4em 5em 6em 5em' }}>
```

Comment:
I'm not sure how to make this one more mobile-friendly besides maybe changing the font size to allow all of the text to fit in. Otherwise, they need to be made of tops and bottoms to allow the text to fit in the container.

```suggestion
      <Typography variant="h5" sx={{ padding: '4em 5em 6em 5em', fontSize: { xs: '0.8rem', sm: '0.9rem', md: '1rem' } }}>
```